docs/content.zh/docs/dev/table/sql/queries/window-tvf.md

```diff
@@ -0,0 +1,264 @@
+
+### TUMBLE
+
+The `TUMBLE` function assigns each element to a window of a specified window size. Tumbling windows have a fixed size and do not overlap. For example, if you specify a tumbling window with a size of 5 minutes, the current window will be evaluated and a new window will be started every five minutes as illustrated by the following figure.
+
+{{< img src="/fig/tumbling-windows.svg" alt="Tumbling Windows" width="70%">}}
+
+The `TUMBLE` function assigns a window for each row of a relation based on a [time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}) column. The return value of `TUMBLE` is a new relation that includes all columns of original relation as well as additional 3 columns named "window_start", "window_end", "window_time" to indicate the assigned window. The original time attribute "timecol" will be a regular timestamp column after window TVF.
+
+`TUMBLE` function takes three required parameters:
+
+```sql
+TUMBLE(TABLE data, DESCRIPTOR(timecol), size)
+```
+
+- `data`: is a table parameter that can be any relation with a time attribute column.
+- `timecol`: is a column descriptor indicating which [time attributes]({{< ref "docs/dev/table/concepts/time_attributes" >}}) column of data should be mapped to tumbling windows.
+- `size`: is a duration specifying the width of the tumbling windows.
+
+Here is an example invocation on the `Bid` table (we simplify `bidtime` only to show the time part):
+
+```sql
+> SELECT * FROM Bid;
+--+---------+-------+------+
+--| bidtime | price | item |
+--+---------+-------+------+
+--| 08:07   | $2    | A    |
+--| 08:11   | $3    | B    |
+--| 08:05   | $4    | C    |
+--| 08:09   | $5    | D    |
+--| 08:13   | $1    | E    |
+--| 08:17   | $6    | F    |
+--+---------+-------+------+
+
+> SELECT * FROM TABLE(
+   TUMBLE(TABLE Bid, DESCRIPTOR(bidtime), INTERVAL '10' MINUTES));
+-- or with the named params
+-- note: the DATA param must be the first
+> SELECT * FROM TABLE(
+   TUMBLE(
+     DATA => TABLE Bid,
+     TIMECOL => DESCRIPTOR(bidtime),
+     SIZE => INTERVAL '10' MINUTES));
+--+---------+-------+------+--------------+------------+--------------+
+--| bidtime | price | item | window_start | window_end | window_time  |
+--+---------+-------+------+--------------+------------+--------------+
+--| 08:07   | $2    | A    | 08:00        | 08:10      | 08:09:59.999 |
+--| 08:11   | $3    | B    | 08:10        | 08:20      | 08:19:59.999 |
+--| 08:05   | $4    | C    | 08:00        | 08:10      | 08:09:59.999 |
+--| 08:09   | $5    | D    | 08:00        | 08:10      | 08:09:59.999 |
+--| 08:13   | $1    | E    | 08:10        | 08:20      | 08:19:59.999 |
+--| 08:17   | $6    | F    | 08:10        | 08:20      | 08:19:59.999 |
+--+---------+-------+------+--------------+------------+--------------+
+
+-- apply aggregation on the tumbling windowed table
+> SELECT window_start, window_end, SUM(price)
```

Comment:
nit： we can use `TIMESTAMP` for `bidtime` and `DECIMAL` for price so that user can reproduce this case easily